latex2sympy.py

```diff
@@ -717,3 +709,34 @@ def get_differential_var_str(text):
     if text[0] == "\\":
         text = text[1:]
     return text
+
+
+class Div(sympy.Mul):
+
+    def __new__(cls, *args, in_parsing=False, **options):
```

Comment:
Actually I'm not really liking the `in_parsing` argument for any of these new classes. I'm not seeing why we would ever want them to do the opposite of what they say they do (and by default).

Reply:
@tragessere 🤔 hm, good point